app/assets/stylesheets/darkswarm/_shop-navigation.css.scss

```diff
@@ -60,17 +60,25 @@ ordercycle {
     }
 
     select {
+      background-image: url('/assets/white-caret.svg');
+    }
+
+    p {
+      text-align: left;
+    }
+
+    select,
+    p {
```

Comment:
We're defining a block for `select`, then for `p`, then for `select` and `p` together...? This can be one block, right?

Reply:
no, one block is only for p, one block is only for select, the last one is for both.
is there a shorter way to do this?

Reply:
Ah, sorry! Right. I think I looked at this before coffee this morning! Never a good idea.